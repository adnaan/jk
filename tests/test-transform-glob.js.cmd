jk transform --stdout -c '({ number }) => ({ plusone: number + 1 })' ./test-transform-files/*.yaml
# This tests that
#  1. all files are processed
#  2. the transformed documents from multidoc files are combined with
#     documents into one multidoc output
