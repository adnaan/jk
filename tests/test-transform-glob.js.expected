plusone: 6
---
plusone: 2
---
plusone: 3
